data/campaigns/Winds_of_Fate/_main.cfg

```diff
@@ -19,7 +19,8 @@
     image="data/core/images/portraits/drakes/inferno.webp~SCALE(350,350)"
     description= _ "As the once plentiful hunting grounds of Morogor slip below the waves, war looms between drakes driven rogue by hunger and those still honorable. Yet the winds of fate may be shifting, for inspired by legends of vast far away lands, a pack of fledgling hunters makes ready to lead their kind on a journey towards salvation.
 
-" + _ "(Hard level, 11 scenarios.)"
+" + _ "(Hard level, 11 scenarios.)
+Features full recall costs and zero gold carryover. Turn limits are rare."
```

Comment:
I assume full recall cost means recall cost equals recruit cost? Not sure if that is obvious.

Reply:
Your assumption is correct, though importantly it is the full cost of a unit at its current advancement level, not its original recruit cost (as a level 1). For example, a Drake Fighter recruited at 17 gold which has advanced to become a Drake Blademaster will cost 62 gold to recall. So the main impression players should be given here is that their recalls will generally cost much more than usual.

Also, if this note is longer than a single line of added text then a confusing (and ugly) scroll bar appears inside the description window. Cutting the sentence about turn limits frees up some space but not by too much, unfortunately.